docker/ravendb-ubuntu/Dockerfile.x64

```diff
@@ -2,28 +2,42 @@ FROM mcr.microsoft.com/dotnet/runtime-deps:7.0-jammy
 
 RUN apt-get update \
     && apt-get install -y \
-    && apt-get install --no-install-recommends openssl bzip2 jq curl -y
-
-ENV RAVEN_ARGS='' RAVEN_SETTINGS='' RAVEN_Setup_Mode='Initial' RAVEN_DataDir='RavenData' RAVEN_ServerUrl_Tcp='38888' RAVEN_AUTO_INSTALL_CA='true' RAVEN_IN_DOCKER='true'
+    && apt-get install --no-install-recommends openssl unzip jq curl libc6-dev -y
+
+ENV RAVEN_ARGS='' \
+    RAVEN_SETTINGS='' \
+    RAVEN_IN_DOCKER='true' \
+    RAVEN_Setup_Mode='Initial' \
+    RAVEN_ServerUrl_Tcp='38888' \
+    RAVEN_AUTO_INSTALL_CA='true' \
+    RAVEN_DataDir='/var/lib/ravendb/data' \
+    RAVEN_Indexing_NugetPackagesPath='/var/lib/ravendb/nuget' \
+    RAVEN_Logs_Path='/var/log/ravendb/logs' \
+    RAVEN_Security_AuditLog_FolderPath='/var/log/ravendb/audit' \
+    RAVEN_Security_MasterKey_Path='/etc/ravendb/security/master.key' \
+    RAVEN_Setup_Certificate_Path='/etc/ravendb/security/server.pfx'
 
 EXPOSE 8080 38888 161
 
-COPY RavenDB.tar.bz2 /opt/RavenDB.tar.bz2
+COPY package-22.04.zip /opt/ravendb-deb.zip
```

Comment:
as soon as the wget is gonna be available, I'll correct that

Reply:
we would like to have ARG for ravendb deb package path if possible. Not zip with a package inside note.
You'll need to adjust powershell build scripts as well.

Reply:
and also propagate to arm64 docker file.